config.js

```diff
@@ -0,0 +1,20 @@
+/*
+  
+This file will be in the codebase and hence public. Add any passwords, 
+session keys and API keys to the .env file.  
+
+*/
+
+
+exports.keystoneInit = {
+  'name': 'nyaaya_cms',
```

Comment:
what style guide are we using?

Reply:
Sublime-HTMLPrettify package; it is one of the most ones on sublime. I am not sure which style guide it follows, ideally it should work on every save; but it was not. Most probably next push will not have style probs.